app/src/main/java/chat/rocket/android/settings/ui/SettingsFragment.kt

```diff
@@ -84,7 +87,22 @@ class SettingsFragment : Fragment(), SettingsView, AdapterView.OnItemClickListen
                 shareIntent.putExtra(Intent.EXTRA_TEXT, shareSub)
                 startActivity(Intent.createChooser(shareIntent, getString(R.string.msg_share_using)))
             }
+            resources.getString(R.string.title_rate_us) ->
+                startAppPlayStore()
+        }
+    }
 
+    fun getMarketAppLink(): String {
```

Comment:
It fits in one line.

` fun getMarketAppLink() = getString(R.string.market_link)`